flink-runtime-web/web-dashboard/src/app/pages/job-manager/profiler/job-manager-profiler.component.html

```diff
@@ -31,6 +31,58 @@
           ></nz-input-number>
         </nz-form-control>
       </nz-form-item>
+      <nz-form-item>
+        <nz-form-label nzFor="mode">Profiling Mode</nz-form-label>
+        <nz-form-control>
+          <nz-select
+            [(ngModel)]="selectMode"
+            nzPlaceHolder="Profiling Mode"
+            name="mode"
+            style="width: 200px"
+          >
+            <nz-option nzCustomContent nzValue="CPU" nzLabel="CPU">
+              <span
+                nz-tooltip
+                nzTooltipTitle="In this mode profiler collects stack trace samples that include Java methods, native calls, JVM code and kernel functions."
+              >
+                CPU
+              </span>
+            </nz-option>
+            <nz-option nzCustomContent nzValue="LOCK" nzLabel="Lock">
+              <span
+                nz-tooltip
+                nzTooltipTitle="In lock profiling mode the top frame is the class of lock/monitor, and the counter is number of nanoseconds it took to enter this lock/monitor."
+              >
+                Lock
+              </span>
+            </nz-option>
+            <nz-option nzCustomContent nzValue="WALL" nzLabel="Wall-Clock">
+              <span
+                nz-tooltip
+                nzTooltipTitle="Wall-Clock option tells async-profiler to sample all threads equally every given period of time regardless of thread status: Running, Sleeping or Blocked. For instance, this can be helpful when profiling application start-up time."
+              >
+                Wall-Clock
+              </span>
+            </nz-option>
+            <nz-option nzCustomContent nzValue="ALLOC" nzLabel="Allocation">
+              <span
+                nz-tooltip
+                nzTooltipTitle="In allocation profiling mode the top frame of every call trace is the class of the allocated object, and the counter is the heap pressure (the total size of allocated TLABs or objects outside TLAB)."
+              >
+                Allocation
+              </span>
+            </nz-option>
+            <nz-option nzCustomContent nzValue="ITIMER" nzLabel="ITIMER">
```

Comment:
I think we can make this mode as the default one or the first one.

Reply:
That makes sense, I have changed the default mode to `ITIMER`.